AnkiDroid/src/main/java/com/ichi2/libanki/Utils.java

```diff
@@ -195,6 +195,9 @@ public static String remainingTime(Context context, long time_s) {
         Resources res = context.getResources();
         if (time_s < TIME_HOUR_LONG) {
             time_x = (int) Math.round(time_s / TIME_MINUTE);
+            if(time_x == 0) {
```

Comment:
nitpick: matches coding style
```suggestion
            if (time_x == 0) {
```